app/Providers/AppServiceProvider.php

```diff
@@ -109,8 +109,14 @@ protected function registerEvents(): void
         // Group edit events
         Event::listen(\App\Events\EditGroup::class, \App\Listeners\EditWordpressPostForGroup::class);
         
-        // Password events
-        Event::listen(\App\Events\PasswordChanged::class, \App\Listeners\ChangeWikiPassword::class);
+        // Password events - only register the wiki-related listeners if wiki integration is enabled
+        if (env('FEATURE__WIKI_INTEGRATION') === true) {
+            Event::listen(\App\Events\PasswordChanged::class, \App\Listeners\ChangeWikiPassword::class);
+        } else {
+            Event::listen(\App\Events\PasswordChanged::class, function() {
+                // Do nothing, wiki integration is disabled
+            });
```

Comment:
I'm assuming something breaks if we just don't listen to the `PassowrdChanged` event when wiki integration is disabled?

Reply:
I confirmed not registering an event listener for the PasswordChange event does not break anything.

Addressed in [9cb7d1f](https://github.com/iFixit/restarters/pull/10/commits/9cb7d1f4703b8e8cb26618acdd02fef0addcbc44)